src/components/PrintModal/PrintModal.js

```diff
@@ -514,35 +521,40 @@ class PrintModal extends React.PureComponent {
                   type="radio"
                   label={t('option.print.all')}
                   defaultChecked
+                  disabled={isProcessingPagesToPrint}
                 />
                 <Input
                   ref={this.currentPage}
                   id="current-page"
                   name="pages"
                   type="radio"
                   label={t('option.print.current')}
+                  disabled={isProcessingPagesToPrint}
                 />
                 <Input
                   ref={this.customPages}
                   id="custom-pages"
                   name="pages"
                   type="radio"
                   label={customPagesLabelElement}
+                  disabled={isProcessingPagesToPrint}
                 />
                 <Input
                   ref={this.includeComments}
                   id="include-comments"
                   name="comments"
                   type="checkbox"
                   label={t('option.print.includeComments')}
+                  disabled={isProcessingPagesToPrint}
                 />
               </form>
             </div>
             {this.state.allowWatermarkModal && (
               <button
                 id="applyWatermark"
                 className="apply-watermark"
-                onClick={() => this.setWatermarkModalVisibility(true)}
+                disabled={isProcessingPagesToPrint}
+                onClick={() => !isProcessingPagesToPrint && this.setWatermarkModalVisibility(true)}
```

Comment:
Can you use a `if` instead of `&&` here? Thanks!

Reply:
done